client/core/core.go

```diff
@@ -3155,8 +3155,8 @@ func handlePenaltyMsg(c *Core, dc *dexConnection, msg *msgjson.Message) error {
 	}
 	t := encode.UnixTimeMilli(int64(note.Penalty.Time) * 1000)
 	d := time.Duration(note.Penalty.Duration)
-	details := fmt.Sprintf("Penalty from DEX at %s\nbroken rule: %s\ntime: %v\nduration: %v\ndetails: \"%s\"\n",
-		dc.acct.host, note.Penalty.Rule, t, d, note.Penalty.Details)
+	details := fmt.Sprintf("Penalty from DEX at %s\nbroken rule: %s\ntime: %v\nduration: %v\naccountID: %v\ndetails: \"%s\"\n",
+		dc.acct.host, note.Penalty.Rule, t, d, note.Penalty.AccountID, note.Penalty.Details)
```

Comment:
Since we have this, and it's part of the data signed by the DEX, let's confirm that `note.Penalty.AccountID` matches the client's account ID with the DEX.

Reply:
Good idea! Will do.